LibreHardwareMonitorLib/Hardware/Sensor.cs

```diff
@@ -1,7 +1,7 @@
 // This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
 // If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
 // Copyright (C) LibreHardwareMonitor and Contributors.
-// Partial Copyright (C) Michael M�ller <mmoeller@openhardwaremonitor.org> and Contributors.
+// Partial Copyright (C) Michael M�ller <mmoeller@openhardwaremonitor.org> and Contributors.
```

Comment:
Undo

Reply:
Hmmm... Yeah, i'm concerned my IDE is using the wrong codepage.